### PR TITLE
Redirect

### PR DESCRIPTION
**Pull request by pieterDekker**
Ready to merge into development, please confirm

### CODE REVIEW

File: sampleExtension/background.js [hunk 1]
```diff
@@ -10,26 +17,25 @@ updateStorage = function() {
 
 // This function receives the blacklist from the sync storage.
 updateBlockedSites = function(callback){
-    getStorageBlacklist(function(blacklist) {
```
Comment (Mantsje) [inline sampleExtension/background.js]: keep the red part here please

File: sampleExtension/background.js [hunk 2]
```diff
@@ -10,26 +17,25 @@ updateStorage = function() {
...
+    getStorageBlacklist(function(result) {
+        blockedSites = result;
         return callback();
     });
 };
 
 // This function adds one url to the blacklist
 addToBlockedSites = function(urlToAdd) {
-    var blockedSiteItem = new BlockedSite(formatUrl(urlToAdd));
-    blockedSites.push(blockedSiteItem);
```
Comment (Mantsje) [inline sampleExtension/background.js]: keep the red part here please

File: sampleExtension/background.js [hunk 3]
```diff
@@ -10,26 +17,25 @@ updateStorage = function() {
...
     updateStorage();
-    replaceListener();
 };
 
 formatUrl = function(url) {
     result = url.split(['//'])[1];
     result = result.split("").reverse().join("");
     result = result.split(['/'])[1];
     result = result.split("").reverse().join("");
-    return result;
```
Comment (Mantsje) [inline sampleExtension/background.js]: keep the red part here please

File: sampleExtension/background.js [hunk 4]
```diff
@@ -10,26 +17,25 @@ updateStorage = function() {
...
 };
 
 // This function adds one url to the blacklist
 addToBlockedSites = function(urlToAdd) {
-    var blockedSiteItem = new BlockedSite(formatUrl(urlToAdd));
-    blockedSites.push(blockedSiteItem);
+    var formattedUrl = formatUrl(urlToAdd);
+    blockedSites.push(formattedUrl);
     updateStorage();
-    replaceListener();
```
Comment (Mantsje) [inline sampleExtension/background.js]: keep the red part here please

File: sampleExtension/manifest.json [hunk 5]
```diff
@@ -22,15 +22,15 @@
   },
 
   "background": {
-    "scripts": ["jquery/jquery-1.10.2.js", "blockedSite.js", "syncStorage.js", "background.js", "init.js"]
```
Comment (Mantsje) [inline sampleExtension/manifest.json]: combine these 2 lines please

File: sampleExtension/background.js (hunk 1 above)
Comment (pieterDekker) [inline sampleExtension/background.js]: check

File: sampleExtension/background.js (hunk 2 above)
Comment (pieterDekker) [inline sampleExtension/background.js]: check

File: sampleExtension/background.js (hunk 4 above)
Comment (pieterDekker) [inline sampleExtension/background.js]: check

File: sampleExtension/background.js (hunk 3 above)
Comment (pieterDekker) [inline sampleExtension/background.js]: check

File: sampleExtension/manifest.json (hunk 5 above)
Comment (pieterDekker) [inline sampleExtension/manifest.json]: check

File: sampleExtension/background.js [hunk 6]
```diff
@@ -76,8 +107,14 @@ addBrowserActionListener = function() {
     });
 };
 
-
-
+addSkipMessageListener = function() {
+    chrome.runtime.onMessage.addListener(function(request, sender) {
+        if (request.message == "goToOriginalDestination") {
+            console.log("intercept set to " + doIntercept + ", going to" + request.destination);
```
Comment (spazzy) [inline sampleExtension/background.js]: console.log's should probably be removed.

File: sampleExtension/manifest.json [hunk 7]
```diff
@@ -22,15 +22,15 @@
...
   },
 
   //TODO update to current state of extension
   "content_scripts": [
     {
-      "matches": ["<all_urls>"],
+      "matches" : ["https://zeeguu.herokuapp.com/get-ex"],
       "css": ["jquery/jquery-ui.css"],
-      "js": ["jquery/jquery-1.10.2.js"]
+      "js" : ["jquery/jquery-1.10.2.js","constants.js","syncStorage.js", "intercept/inject.js"]
```
Comment (spazzy) [inline sampleExtension/manifest.json]: aren't constants.js and syncStorage.js loaded twice this way? Once in the background and once in the content?

Comment (pieterDekker) [inline sampleExtension/manifest.json]: yes, but they are required in both

**Pull request closed without merging**